modelexpress_client/python/modelexpress/weight_transfer/protocol/__init__.py

```
@@ -0,0 +1,45 @@
+    "ResolvedRegion",
+    "RdmaDescriptor",
+    "SyncMode",
+    "encode_trainer_table",
+    "decode_trainer_table",
+    "encode_inference_table",
+    "decode_inference_table",
+    "encode_resolved_regions",
+    "decode_resolved_regions",
+    "encode_rdma_descriptors",
+    "decode_rdma_descriptors",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)